scripts/init-submodules-no-riscv-tools.sh

```diff
@@ -6,14 +6,27 @@ set -o pipefail
 
 unamestr=$(uname)
 RDIR=$(pwd)
+scripts_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
 # ignore riscv-tools for submodule init recursive
 # you must do this globally (otherwise riscv-tools deep
 # in the submodule tree will get pulled anyway
 git config --global submodule.riscv-tools.update none
 git config --global submodule.esp-tools.update none
 git config --global submodule.experimental-blocks.update none
```

Comment:
So just to make sure I understand correctly, the user must init the rebar submodules first, then they run the firesim init script. How do we want to document this (put this in the rebar docs? and firesim docs) or does FireSim handle submodule init differently?

Reply:
Yeah, that's correct. I figure that in many cases people won't want to use firesim (they might be doing local dev, for instance). 

REBAR docs; build-setup.sh has a different flow when initializing firesim-as-top.